qemu/tests/qmp_command.py

```diff
@@ -262,10 +245,6 @@ def send_cmd(cmd):
     cmd_return_value = params.get("cmd_return_value")
     exception_list = params.get("exception_list", "")
 
-    # HOOKs
-    if result_check == 'qmp_cpu':
-        pre_cmd = "cpu index=%d" % (int(params['smp']) - 1)
-
```

Comment:
It can, i t is only a input param for the function that used to compare.

 (1/1) smp_4.4096m.repeat1.run_test.Host_RHEL.m7.u4.qcow2.virtio_blk.up.virtio_net.RHEL.7.4.x86_64.io-github-autotest-qemu.qmp_command.qmp_cpu: PASS (46.44 s)